aep/general/0231/aep.md.j2

```diff
@@ -58,8 +73,8 @@ message BatchGetBooksRequest {
 }
 ```
 
-- The request and response schemas **must** match the method name, with `Request` and
-  `Response` suffixes.
+- The request and response schemas **must** match the method name, with
+  `Request` and `Response` suffixes.
```

Comment:
Is this proto-specific?

Reply:
Only the request part. I'll move the response part to that section and make it common.